### ISSUE TITLE
Pre-built binary possibly not found

### ISSUE THREAD

**Issue by romgrk** [romgrk/node-gtk#73]
```
node-pre-gyp WARN Pre-built binaries not installable for node-gtk@0.2.0 and node@10.6.0 (node-v64 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error getaddrinfo ENOTFOUND node-gtk-1.s3-us-east-1.amazonaws.com node-gtk-1.s3-us-east-1.amazonaws.com:443 
```

**Comment by ledlamp** [comment]
Yeah the domain node-gtk-1.s3-us-east-1.amazonaws.com doesn't exist (doesn't resolve to anything)

**Comment by romgrk** [comment]
Fixed in bf70f88780e38910b87e0190ae1b020aaa848ed9. I'm not sure how I fixed it. If anyone runs in this issue, don't hesitate to re-open.
